### PR TITLE
Intro to HTML and CSS lesson: Fix first paragraph about selectors 

### LINKED ISSUE #25719: <Intro to CSS>: <Reorganizing structure of explanations>
### Describe your suggestion

I was looking through the ID and Class portions of the Selectors lesson. I think it would be beneficial to put this statement "The major difference between classes and IDs is that an element can only have one ID. An ID cannot be repeated on a single page, and the ID attribute should not contain any whitespace at all."  in the beginning paragraph. Here's what I mean:

* ID selectors are similar to class selectors. They select an element with the given ID, which is another attribute you place on an HTML element. The major difference between classes and IDs is that an element can only have one ID. It cannot be repeated on a single page and should not contain any whitespace. 

By doing this, the explanation would be clearer to learners and they know how to use it effectively early on. 

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-intro-to-css

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### (Optional) Discord Name

Chrissy

### (Optional) Additional Comments

I just want to say that's been awesome contributing to the curriculum, and I can't wait to think of more ways to continue its improvement.

### PR DESCRIPTION
**Pull request by CBID2**
## Because

I created this PR to improve the lesson on selectors.

## This PR

This PR places the statement "The major difference between classes and IDs is that an element can only have one ID. An ID cannot be repeated on a single page, and the ID attribute should not contain any whitespace at all." in the beginning paragraph on selectors. 

## Issue 

Closes #25719

## Additional Information

The Markdown Lint extension pointed out some errors in other areas of the lesson. I tried fixing them to no avail, so I would some assistance on this.  
## Pull Request Requirements

-   [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [X] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [X] The `Because` section summarizes the reason for this PR
-   [X] The `This PR` section has a bullet point list describing the changes in this PR
-   [X] If this PR addresses an open issue, it is linked in the `Issue` section
-   [X] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [X] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by ManonLef** [comment]
Hi @CBID2 thank for taking this on. Could you please change the formatting back to adhere to our [Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#lesson-layout)?

**Comment by CBID2** [comment]
> Hi @CBID2 thank for taking this on. Could you please change the formatting back to adhere to our [Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#lesson-layout)?

Sure

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-### Introduction
+# Introduction
```
Comment (ManonLef) [inline foundations/html_css/css-foundations/intro-to-css.md:1]: needs to be changed back to adhere to style guide

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
 In the previous lesson, you learned how to write the HTML that determines how a web page is structured. The next step is to make that structure look good with some _style_, which is exactly what CSS is for. In the next few lessons, we're going to focus on what we believe are some foundational CSS concepts, things that everyone should know from the beginning — whether they are just starting out or simply need a refresher.
 
-### Lesson Overview
+## Lesson Overview
```
Comment (ManonLef) [inline foundations/html_css/css-foundations/intro-to-css.md:5]:
```suggestion
### Lesson Overview
```

change needs to be reverted to adhere to style guide

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 3]
```diff
@@ -405,10 +399,10 @@ This section contains questions for you to check your understanding of this less
 
 - <a class="knowledge-check-link" href="#class-selectors">What is the syntax for class and ID selectors?</a>
 - <a class="knowledge-check-link" href="#the-grouping-selector">How would you apply a single rule to two different selectors?</a>
-- <a class="knowledge-check-link" href="#chaining-selectors">Given an element that has an id of `title` and a class of `primary`, how would you use both attributes for a single rule?</a>
+- <a class="knowledge-check-link" href="#chaining-selectors"> Given an element that has an id of `title` and a class of `primary`, how would you use both attributes for a single rule?</a>
```
Comment (ManonLef) [inline foundations/html_css/css-foundations/intro-to-css.md:402]: I'm not entirely sure why there are two edits in this section for two knowledge checks, but the same edits were not made for the other checks. I think this is unrelated to the PR/issue so could safely be reverted back unless I'm missing something, please do tell me if so.

**Review by ManonLef — CHANGES_REQUESTED** [review]
I already requested changes to revert the headings back to adhere to the style guide. I noticed some other changes that I noted in the review for the knowledge check section. .

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 4]
```diff
@@ -84,13 +81,13 @@ Class selectors will select all elements with the given class, which is just an
 }
 ~~~
 
-Note the syntax for class selectors: a period immediately followed by the case-sensitive value of the class attribute. Classes aren't required to be specific to a particular element, so you can use the same class on as many elements as you want.
+Note The syntax for class selectors: a period immediately followed by the case-sensitive value of the class attribute. Classes aren't required to be specific to a particular element, so you can use the same class on as many elements as you want.
```
Comment (ManonLef) [inline foundations/html_css/css-foundations/intro-to-css.md:84]:
```suggestion
Note the syntax for class selectors: a period immediately followed by the case-sensitive value of the class attribute. Classes aren't required to be specific to a particular element, so you can use the same class on as many elements as you want.
```

File: foundations/html_css/css-foundations/intro-to-css.md (hunk 3 above)
Comment (CBID2) [reply foundations/html_css/css-foundations/intro-to-css.md]: Hey, @ManonLef I made the reverts. The comment is unrelated to this PR. I was making a mere observation on formatting. Should I raise a related issue about the formatting? I know TOP has a standard but it can be very confusing for contributors when markdown: lint points them out as errors.

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 5]
```diff
@@ -106,9 +103,7 @@ ID selectors are similar to class selectors. They select an element with the giv
 }
 ~~~
 
-Instead of a period, we use a hashtag immediately followed by the case-sensitive value of the ID attribute. A common pitfall is people overusing the ID attribute when they don't necessarily need to, and when classes will suffice. While there are cases where using an ID makes sense or is needed, such as taking advantage of specificity or having links redirect to a section on the current page, you should use IDs **sparingly** (if at all).
-
-The major difference between classes and IDs is that an element can only have **one** ID. An ID cannot be repeated on a single page, and the ID attribute should not contain any whitespace at all.
+Also, instead of a period, we use a hashtag immediately followed by the case-sensitive value of the ID attribute. A common pitfall is people overusing the ID attribute when they don't necessarily need to, and when classes will suffice. While there are cases where using an ID makes sense or is needed, such as taking advantage of specificity or having links redirect to a section on the current page, you should use IDs **sparingly** (if at all).
```
Comment (ManonLef) [inline foundations/html_css/css-foundations/intro-to-css.md:106]:
```suggestion
For IDs, instead of a period, we use a hashtag immediately followed by the case-sensitive value of the ID attribute. A common pitfall is people overusing the ID attribute when they don't necessarily need to, and when classes will suffice. While there are cases where using an ID makes sense or is needed, such as taking advantage of specificity or having links redirect to a section on the current page, you should use IDs **sparingly** (if at all).
```

After the code block the `also` didn't sound entirely right to me. But looking at the old version I thought it could be improved as well. What do you think of this change?

**Review by ManonLef — CHANGES_REQUESTED** [review]

File: foundations/html_css/css-foundations/intro-to-css.md (hunk 3 above)
Comment (ManonLef) [reply foundations/html_css/css-foundations/intro-to-css.md]: @CBID2 I think you can open an issue so the maintainers more versed in the style guide can chime in. However, I think it's safest to not let any linter or formatter edit files for projects that have a style guide.

File: foundations/html_css/css-foundations/intro-to-css.md (hunk 5 above)
Comment (CBID2) [reply foundations/html_css/css-foundations/intro-to-css.md]: I like it! :) The explanation is clearer now

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 6]
```diff
@@ -13,8 +13,6 @@ This section contains a general overview of topics that you will learn in this l
 For a more interactive explanation and example, try the following Scrim (let us know what you think of these):
 
 <iframe src="https://scrimba.com/scrim/co12d4cf99cf2776f19e84a9d?embed=odin,mini-header,no-sidebar,no-next-up" sandbox="allow-scripts allow-same-origin allow-popups" width="100%" height="400"></iframe>
-
-
 ### Basic Syntax
```
Comment (ManonLef) [inline foundations/html_css/css-foundations/intro-to-css.md:16]: there's an empty line needed above this header. Please make sure to check all sections that changed with our [preview tool](https://www.theodinproject.com/lessons/preview)

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 7]
```diff
@@ -23,10 +21,9 @@ At the most basic level, CSS is made up of various rules. These rules are made u
 
 <div class="lesson-note" markdown="1">
 
-#### Note
-
-A `<div>` is one of the basic HTML elements. It is simply an empty container. In general, it is best to use other tags such as `<h1>` or `<p>` for content in your projects, but as we learn more about CSS you'll find that there are many cases where the thing you need is just a container for other elements. Many of our exercises use plain `<div>`s for simplicity. Later lessons will go into much more depth about when it is appropriate to use the various HTML elements.
-
+ #### Note
```
Comment (ManonLef) [inline foundations/html_css/css-foundations/intro-to-css.md:24]: This note and its heading doesn't render well in the lesson preview tool

**Review by ManonLef — CHANGES_REQUESTED** [review]
I'm still noting some issues in the lesson preview tool.

File: foundations/html_css/css-foundations/intro-to-css.md (hunk 7 above)
Comment (CBID2) [reply foundations/html_css/css-foundations/intro-to-css.md:24]: Hmmm... I wonder how I can change it. That's how it was originally

Comment (ManonLef) [reply foundations/html_css/css-foundations/intro-to-css.md:24]: It has indentation on both the header and body. removing that will probably do the trick

Comment (CBID2) [reply foundations/html_css/css-foundations/intro-to-css.md:26]: Ok

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 8]
```diff
@@ -23,10 +22,10 @@ At the most basic level, CSS is made up of various rules. These rules are made u
 
 <div class="lesson-note" markdown="1">
 
+
```
Comment (ManonLef) [inline foundations/html_css/css-foundations/intro-to-css.md:25]: .

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 9]
```diff
@@ -23,10 +22,10 @@ At the most basic level, CSS is made up of various rules. These rules are made u
 
 <div class="lesson-note" markdown="1">
 
+ 
 #### Note
-
-A `<div>` is one of the basic HTML elements. It is simply an empty container. In general, it is best to use other tags such as `<h1>` or `<p>` for content in your projects, but as we learn more about CSS you'll find that there are many cases where the thing you need is just a container for other elements. Many of our exercises use plain `<div>`s for simplicity. Later lessons will go into much more depth about when it is appropriate to use the various HTML elements.
-
+ 
+ A `<div>` is one of the basic HTML elements. It is simply an empty container. In general, it is best to use other tags such as `<h1>` or `<p>` for content in your projects, but as we learn more about CSS you'll find that there are many cases where the thing you need is just a container for other elements. Many of our exercises use plain`<div>`s for simplicity. Later lessons will go into much more depth about when it is appropriate to use the various HTML elements.
```
Comment (ManonLef) [inline foundations/html_css/css-foundations/intro-to-css.md:28]:
```suggestion
A `<div>` is one of the basic HTML elements. It is simply an empty container. In general, it is best to use other tags such as `<h1>` or `<p>` for content in your projects, but as we learn more about CSS you'll find that there are many cases where the thing you need is just a container for other elements. Many of our exercises use plain`<div>`s for simplicity. Later lessons will go into much more depth about when it is appropriate to use the various HTML elements.

```

File: foundations/html_css/css-foundations/intro-to-css.md (hunk 8 above)
Comment (CBID2) [reply foundations/html_css/css-foundations/intro-to-css.md:25]: Umm...what does that mean?

Comment (ManonLef) [reply foundations/html_css/css-foundations/intro-to-css.md:25]: I'm sorry, that comment was for me, hoping I could edit and commit that section myself and get this merge through. 
Could you please remove the extra line below the opening div for the note. There are currently two new lines, it should only be one.

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 10]
```diff
@@ -23,9 +22,10 @@ At the most basic level, CSS is made up of various rules. These rules are made u
 
 <div class="lesson-note" markdown="1">
 
+
```
Comment (ManonLef) [inline foundations/html_css/css-foundations/intro-to-css.md:25]:
```suggestion
```

File: foundations/html_css/css-foundations/intro-to-css.md (hunk 8 above)
Comment (CBID2) [reply foundations/html_css/css-foundations/intro-to-css.md:25]: Sure

**Review by ManonLef — COMMENTED** [review]
I can do this @CBID2 so don't worry about it

**Comment by ManonLef** [comment]
Thank you

**Review by ManonLef — APPROVED** [review]

File: foundations/html_css/css-foundations/intro-to-css.md (hunk 8 above)
Comment (CBID2) [reply foundations/html_css/css-foundations/intro-to-css.md]: Done

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 11]
```diff
@@ -24,10 +23,15 @@ At the most basic level, CSS is made up of various rules. These rules are made u
...
+ 
+<<<<<<< HEAD
+ A `<div>` is one of the basic HTML elements. It is simply an empty container. In general, it is best to use other tags such as `<h1>` or `<p>` for content in your projects, but as we learn more about CSS you'll find that there are many cases where the thing you need is just a container for other elements. Many of our exercises use plain`<div>`s for simplicity. Later lessons will go into much more depth about when it is appropriate to use the various HTML elements.
+ </div>
+=======
+A `<div>` is one of the basic HTML elements. It is simply an empty container. In general, it is best to use other tags such as `<h1>` or `<p>` for content in your projects, but as we learn more about CSS you'll find that there are many cases where the thing you need is just a container for other elements. Many of our exercises use plain`<div>`s for simplicity. Later lessons will go into much more depth about when it is appropriate to use the various HTML elements.
```
Comment (ManonLef) [inline foundations/html_css/css-foundations/intro-to-css.md:31]:
```suggestion
A `<div>` is one of the basic HTML elements. It is simply an empty container. In general, it is best to use other tags such as `<h1>` or `<p>` for content in your projects, but as we learn more about CSS you'll find that there are many cases where the thing you need is just a container for other elements. Many of our exercises use plain`<div>`s for simplicity. Later lessons will go into much more depth about when it is appropriate to use the various HTML elements.
```

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 12]
```diff
@@ -24,10 +23,15 @@ At the most basic level, CSS is made up of various rules. These rules are made u
...
 
 </div>
+>>>>>>> refs/remotes/origin/reorganizing-section-on-CSS-selectors
```
Comment (ManonLef) [inline foundations/html_css/css-foundations/intro-to-css.md:34]:
```suggestion
```
